package.json

```diff
@@ -20,8 +20,9 @@
 		"socket.io": "*"
 	},
 	"engines": {
-		"node": "8"
+		"node": ">= 6"
```

Comment:
Per discussion in #40 we settled on a minimum version of 6. I do know that 8.x works as well, so went with >=.